### ISSUE TITLE
Autocomplete bug in VoiceOver on MacOS

### ISSUE THREAD

**Issue by plousia** [adamsilver/nostyle#23]
To clarify, I am not using this code myself, but I recommend the autocomplete pattern to accessibility audit clients. While testing the [demo](https://nostyle.onrender.com/examples/autocomplete) using VoiceOver + keyboard navigation on MacOS, I note a bug: VoiceOver says to use Command + Option + Space to open the combobox and view the options, but this does not work; the list opens briefly and immediately closes. This key pattern combination does work in [this W3 autocomplete example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/).
